modules/edge-identity/src/identity.rs

```diff
@@ -149,70 +158,29 @@ decl_module! {
 			Ok(())
 		}
 
-		/// Verify an existing identity based on its attested proof. Sender
-		/// be specified on the pre-selected list of verifiers.
-		pub fn verify(origin, identity_hash: T::Hash, vote: bool) -> Result {
+		/// Propose verification to be voted upon by the council
+		pub fn verify_or_deny(origin, identity_hash: T::Hash, approve: bool, verifier_index: usize) -> Result {
 			let _sender = ensure_signed(origin)?;
-
-			ensure!(Self::verifiers().contains(&_sender), "Sender not a verifier");
+			ensure!(verifier_index < Self::verifiers().len(), "Verifier index out of bounds");
```

Comment:
If you wanted to do the council thing, then this is where you'd use the ApproveOrigin (or RejectOrigin) calls to validate the sender.